### PR TITLE
Fix no-op unit tests

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
Fix tests introduced in https://github.com/kubevirt/kubevirt/pull/10477

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-handler/heartbeat/ksm_test.go [hunk 1]
```diff
@@ -123,15 +125,18 @@ var _ = Describe("KSM", func() {
...
+		Expect(node.Labels).To(HaveKeyWithValue(kubevirtv1.KSMEnabledLabel, "false"))
 
-		err := os.WriteFile(filepath.Join(fakeSysKSMDir, "run"), []byte("1\n"), 0644)
+		err = os.WriteFile(filepath.Join(fakeSysKSMDir, "run"), []byte("1\n"), 0644)
 		Expect(err).ToNot(HaveOccurred())
 
 		createCustomMemInfo(false)
 
-		testutils.ExpectNodePatch(fakeClient, kubevirtv1.KSMEnabledLabel)
+		// TODO: What is this supposed to do?
```
Comment (jean-edouard) [inline pkg/virt-handler/heartbeat/ksm_test.go:139]: Now that KSM is "running" on the node, the virt-handler heartbeat should set `KSMEnabledLabel` to `true`.

**Review by jean-edouard — COMMENTED** [review]
Looks good, thank you for fixing this!
See in-line comment for the TODO.

File: pkg/virt-handler/heartbeat/ksm_test.go (hunk 1 above)
Comment (xpivarc) [reply pkg/virt-handler/heartbeat/ksm_test.go:139]: Thanks, I have tried but this seems to fail. I am not sure I am going to have time this week to further hunt this.

**Comment by jean-edouard** [comment]
/approve
As discussed offline, the TODO relates to an incomplete annotation check in the code.
Feel free to fix it here, if not I will open a follow-up PR once this one merges.
Thank you!

**Pull request merged**